elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ProviderAttributeDefinitions.java

```diff
@@ -218,10 +240,41 @@ class ProviderAttributeDefinitions {
                     .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
                     .build();
 
+    protected static final SimpleAttributeDefinition AUTHENTICATION_REQUEST_FORMAT =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.AUTHENTICATION_REQUEST_FORMAT, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setDefaultValue(new ModelNode(OAUTH2))
+                    .setAllowedValues(OAUTH2, REQUEST, REQUEST_URI)
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .build();
+
+    protected static final SimpleAttributeDefinition REQUEST_OBJECT_ENCRYPTION_ALGORITHM =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.REQUEST_OBJECT_ENCRYPTION_ALGORITHM, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .setRequires(ElytronOidcDescriptionConstants.REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM)
+                    .build();
+
+    protected static final SimpleAttributeDefinition REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .setRequires(ElytronOidcDescriptionConstants.REQUEST_OBJECT_ENCRYPTION_ALGORITHM)
+                    .build();
+
+    protected static final SimpleAttributeDefinition REQUEST_OBJECT_SIGNING_ALGORITHM =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.REQUEST_OBJECT_SIGNING_ALGORITHM, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setDefaultValue(new ModelNode("none")) // plaintext jwt to be sent
```

Comment:
We should use the Elytron Oidc constant instead of the string "none".

Reply:
Since I dont have them in OIdc anymore, can I create a constant in ElytronOidcDescriptionConstants class instead?